app/src/main/java/com/epfl/beatlink/ui/profile/OtherProfile.kt

```diff
@@ -27,6 +27,12 @@ fun OtherProfileScreen(
     navigationAction: NavigationActions,
     spotifyApiViewModel: SpotifyApiViewModel
 ) {
+  // Updates the sent/received friend requests instantly by reloading the screen
+  LaunchedEffect(Unit) {
+    friendRequestViewModel.getFriendRequests()
+    friendRequestViewModel.getAllFriends()
+  }
+
```

Comment:
I'm not sure whether this is redundant as we already call `getOtherProfileAllFriends` in the ProfileColumn composable when `selectedUserUserId` gets updated.